share/models/core.py

```diff
@@ -183,9 +186,9 @@ def store_data(self, doc_id, data, source, app_label):
         )
 
         if created:
-            logger.debug('Newly created RawData for document {} from {}'.format(doc_id, source))
+            logger.debug('Newly created RawData for document %s from %s', doc_id, source)
```

Comment:
Is there a benefit to using `%s` and logger args? Should we be doing that everywhere?